### PR TITLE
kids homework fix - Jon Prosser

### PR DESCRIPTION
**Pull request by jip1984**
# Your Details

- Your Name:
- Your City:
- Your Slack Name:

# Homework Details

- Module:
- Week:

### CODE REVIEW

File: homework.md [hunk 1]
```diff
@@ -44,70 +44,71 @@ Using CSS, how would I select every `<h1>` tag on the page?
...
 
-To record what changes you made to the code
+To show other developers what you have changed and why. This helps to stop confusion.
 
 ## Question 18
 
 Where can I look to find help with Git?
 
 The Git Cheatsheet!
+
```
Comment (lish-dev) [inline homework.md:114]: Maybe look into adding another suggestion to find help with GIT?

Comment (jip1984) [reply homework.md:114]: Thanks will do

**Review by SenaitEkubai — COMMENTED** [review]
I think you need to add your details to the file to complete the homework.

**Comment by jip1984** [comment]
Thanks for the feedback guys much appreciated.

**Pull request closed without merging**